src/category_theory/preadditive/eilenberg_moore.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2022 Julian Kuelshammer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Julian Kuelshammer
+-/
+
+import category_theory.preadditive.default
+import category_theory.monad.algebra
+import category_theory.preadditive.additive_functor
+
+/-!
+# Preadditive structure on algebras over a monad
+
+If `C` is a preadditive categories and `T` is an additive monad on `C` then `algebra T` is also
+preadditive. Dually, if `U` is an additive comonad on `C` then `coalgebra U` is preadditive as well.
+
+-/
+
+universes v₁ u₁ -- morphism levels before object levels. See note [category_theory universes].
+
+namespace category_theory
+variables (C : Type u₁) [category.{v₁} C] [preadditive C] (T : monad C)
+  [functor.additive (T : C ⥤ C)]
+
+open category_theory.limits preadditive
+
+/-- The category of algebras over an additive monad on a preadditive category is preadditive. -/
+instance monad.algebra_preadditive : preadditive (monad.algebra T) :=
```

Comment:
```suggestion
@[simps]
instance monad.algebra_preadditive : preadditive (monad.algebra T) :=
```
If you have reason to believe that things like `(α + β).f = α.f + β.f` shouldn't be `simp` lemmas, then I think you should add the lemmas by hand so that they are at least available for rewriting.